src/lib/relay/QueryRenderers.tsx

```diff
@@ -492,22 +492,29 @@ export const CitySectionListRenderer: React.SFC<CitySectionListProps> = ({ rende
     partnerType?: PartnerShowPartnerType
     status?: EventStatus
     dayThreshold?: number
+    sort: string
   } = { citySlug }
 
   switch (section) {
     case "museums":
       variables.partnerType = "MUSEUM"
+      variables.status = "RUNNING"
+      variables.sort = "PARTNER_ASC"
       break
     case "galleries":
       variables.partnerType = "GALLERY"
+      variables.status = "RUNNING"
+      variables.sort = "PARTNER_ASC"
       break
     case "closing":
       variables.status = "CLOSING_SOON"
-      variables.dayThreshold = 7
+      variables.sort = "END_AT_ASC"
+      variables.dayThreshold = 30
       break
     case "opening":
       variables.status = "UPCOMING"
-      variables.dayThreshold = 7
+      variables.sort = "START_AT_ASC"
```

Comment:
I don't think this `sort` parameter does anything, since there's no `sort` parameter defined in the GraphQL query. We'll need to add it there for this to work.

Reply:
hey yeah these were all just quick notes essentially on what we need to do. I added it in another commit. About to push it up, but still working on some other small details.

Reply:
Still no sort param on `CitySectionList_city`?

Reply:
hey @ashfurrow it's been added now